File: 1010/1010.c
#include <stdio.h>

int main() {
    int code[2],unit[2];
    double price[2],sum=0.0;
    for(int i=0; i<2; i++)
    {
        scanf("%d %d %lf",&code[i],&unit[i],&price[i]);
        sum+=unit[i]*price[i];
    }
    printf("VALOR A PAGAR: R$ %.2lf\n",sum);
    return 0;
}

File: Python/1010.py
code,unit,price=([] for i in range(3))
for i in range(2):
    ca, ua, pa=input().split()
    code.append(int(ca))
    unit.append(int(ua))
    price.append(float(pa))

sum=0.0
for i in range(2):
    sum+=unit[i]*price[i]

print(f"VALOR A PAGAR: R$ {sum:.2f}")